auth/authlink/store.go

```diff
@@ -53,7 +53,7 @@ type Status struct {
 	AuthedAt   time.Time
 }
 
-const codeAlphabet = "ABCDEFGHJKLMNPQRSTWXYZ23456789"
+const codeAlphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
```

Comment:
It was intentional to completely leave out any symbols that could be confused depending on fonts: Il1, O0. U and V are probably different enough but were left out as well intentionally.

Since we can't easily control the fonts available (esp. in browsers) the thought was to just leave it out of scope and resolve the ambiguity by only using characters that are always visually distinguishable. As long as both devices use the same font we should be okay -- but if we add in the other characters we'll need to add finding the right font -- something that we can instead do at a later time without consequence.

Reply:
I see, I added them back in because:
1. The verify code is now rendered to the user instead of needing to be typed, and
2. The claim code is used programmatically

My thought was that we originally had them left out because of human error in keying in the code, but if there is concern with font as well we can definitely keep those characters out.